doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -128,36 +128,40 @@ Update ACRN Hypervisor Image
    .. code-block:: none
 
       $ sudo lspci -vv
-      00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
-              Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
-      00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
-              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
-      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
-              Subsystem: Intel Corporation Ethernet Connection I219-LM
+      00:14.0 USB controller: Intel Corporation Device 9ded (rev 30) (prog-if 30 [XHCI])
+              Subsystem: Intel Corporation Device 7270
+      00:17.0 SATA controller: Intel Corporation Device 9dd3 (rev 30) (prog-if 01 [AHCI 1.0])
+              Subsystem: Intel Corporation Device 7270
+      02:00.0 Non-Volatile memory controller: Intel Corporation Device f1a8 (rev 03) (prog-if 02 [NVM Express])
+              Subsystem: Intel Corporation Device 390d
+      03:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
+              Subsystem: Intel Corporation I210 Gigabit Network Connection
+      04:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
+              Subsystem: Intel Corporation I210 Gigabit Network Connection
 
    .. note::
-      Verify the PCI devices BDF defined in the
-      ``hypervisor/arch/x86/configs/whl-ipc-i5/pci_devices.h``
+      Verify the ``pci_devs`` defined for VM0 and VM1 in the
+      ``misc/config_tools/data/whl-ipc-i7/logical_partition.xml``
       with the information reported by the ``lspci -vv`` command.
 
 #. Clone the ACRN source code and configure the build options.
 
    Refer to :ref:`getting-started-building` to set up the ACRN build
    environment on your development workstation.
 
-   Clone the ACRN source code and check out to the tag v2.3:
+   Clone the ACRN source code and check out to the tag v2.4:
 
    .. code-block:: none
 
       $ git clone https://github.com/projectacrn/acrn-hypervisor.git
       $ cd acrn-hypervisor
-      $ git checkout v2.3
+      $ git checkout v2.4
 
    Build the ACRN hypervisor and ACPI binaries for pre-launched VMs with default xmls:
 
    .. code-block:: none
 
-      $ make hypervisor BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/whl-ipc-i5.xml SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/whl-ipc-i5/logical_partition.xml RELEASE=0
+      $ make hypervisor BOARD_FILE=whl-ipc-i7 SCENARIO_FILE=logical_partition RELEASE=0
```

Comment:
The use of `BOARD_FILE` and `SCENARIO_FILE` is deprecated, just use `BOARD` and `SCENARIO` directly.

```suggestion
      $ make hypervisor BOARD=whl-ipc-i7 SCENARIO=logical_partition RELEASE=0
```